src/test/scala/com/github/tkawachi/doctest/TestGenSpec.scala

```diff
@@ -269,6 +271,9 @@ object TestGenSpec extends TestSuite {
             |    if (s.headOption == Some('\n')) s.tail else s
             |  }
             |
+            |  implicit def toVoid[A](ref: A)(implicit location: _root_.minitest.api.SourceLocation): _root_.minitest.api.Void =
+            |    _root_.minitest.api.Void.UnitRef
```

Comment:
Could you explain the purpose of this implicit? I don't know minitest well.

Reply:
this is similar to what i had to do with Specs2, in sense that minitest's `test` method is defined as 
```scala
def test(name: String)(f: => Void): Unit
```

so when there's an assertion it's implicitly converted to Void, but for tests that don't have assertions (in in this new only code blocks mode) it fails to compile without this implicit
it's sort of a hack to overcome original idea of minitest to prevent tests without assertions, but this whole thing started because author of monix and minitest asked for this mode without assertions :)